src/js/components/stock-movement-wizard/AddItemsPage.jsx

```diff
@@ -8,6 +8,8 @@ import arrayMutators from 'final-form-arrays';
 import Alert from 'react-s-alert';
 import { confirmAlert } from 'react-confirm-alert';
 
+import 'react-confirm-alert/src/react-confirm-alert.css';
```

Comment:
Should not this be added inside our index app component (same goes to rest of the steps from SM)?

Reply:
no, after code splitting it should be added in every place this component is used

Reply:
;/